region/maridia/outer.json

```diff
@@ -6785,6 +6809,35 @@
                   "requires": [ "Gravity" ]
                 }
               ]
+            },
+            {
+              "usedTiles": 33,
+              "framesRemaining": 70,
+              "shinesparkFrames": 0,
+              "openEnd": 2,
+              "strats": [
+                {
+                  "name": "Stutter Water Shine Charge",
+                  "notable": false,
+                  "requires": [ 
+                    "canStutterWaterShineCharge",
+                    "canDodgeWhileShooting",
+                    "canShinechargeMovement",
+                    {"ammo": {"type": "Super", "count": 1}},
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 2,
+                      "physics": ["air"],
+                      "useFrames": 80
+                    }},
+                    {"resetRoom": {
+                      "nodes": [1],
+                      "nodesToAvoid": [2]
+                    }}
```

Comment:
currently the logic is designed to start a cross room strat at the node it is used. In addition to adding this reset room requirement, can we add a gate glitch enabling Samus to get from 2->1?